docs/how_to_knit/dbj_2-color.md

```diff
@@ -19,8 +19,10 @@ Smiley was made from this image:
 ![ ](img/dbj_2-color/Smiley_4_72x30.png)
 
 AYAB takes care of the colors for you, you just give it a black-and-white
-pattern as if you are going to do single bed Fair Isle.  
-Use thin yarns, you’ll be using side-by-side needles on both beds. (for those
+pattern as if you are going to do single bed Fair Isle. To create your own patterns,
+refer to the AYAB manual, section "pattern_image_creation". Make sure the 
+width of the pattern has an even number of pixels (=stitches).  
```

Comment:
AYAB can handle uneven widths also.

Reply:
For DBJ, the Brother KH850 manual p.38 says you have to select an uneven number of needles on the ribber. So I guess if the pattern is uneven, one endrow of the uneven pattern will not be knitted. I did not test, if the AYAB can realy knit DBJ with an uneven number of needles on the ribber, so I stand corrected.

Reply:
Well, AYAB supports even and uneven widths, it just does not care. But to be honest, I have no idea if there is such a requirement for certain knitting techniques.
I propose to remove the sentence for now, @Adrienne200 can perhaps tell us if there is such a requirement for (un)even pattern widths for some knitting techniques :)